mkdocs/themes/readthedocs/messages.pot

```diff
@@ -1,17 +1,15 @@
 # Translations template for MkDocs.
 # Copyright (C) 2021 MkDocs
 # This file is distributed under the same license as the MkDocs project.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
 #
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: MkDocs 1.2.dev1\n"
-"Report-Msgid-Bugs-To: github@mkdocs.org\n"
-"POT-Creation-Date: 2021-03-14 18:09+0100\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"Project-Id-Version: MkDocs 1.2\n"
+"Report-Msgid-Bugs-To: https://github.com/mkdocs/mkdocs/issues\n"
+"POT-Creation-Date: 2021-04-08 22:46+0200\n"
+"PO-Revision-Date: 2021-04-08 22:46+0200\n"
+"Last-Translator: Alexys Jacob @ultrabug\n"
```

Comment:
I expected `PO-Revision-Date` and `Last-Translator` to be blank on the pot file. Or at least to provide a sample template so that translators know what is expected when they edit po files.

Reply:
of course, my mistake sorry; fixed